robot/Cumulus/tests/browser/recurring_donations/create_recurring_donation_and_check_rollups.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Create Recurring Donation And Check Rollups
+    
+    #Create a Recurring Donation from a Contact
+    ${ns} =  Get NPSP Namespace Prefix
+    &{contact} =                 API Create Contact           Email=jjoseph@robot.com
+    Go To Record Home            &{contact}[Id]
+    &{recurringdonation} =       API Create Recurring Donation  npe03__Contact__c=&{contact}[Id]
```

Comment:
I see u r removing the existing recurring donations test case, in that case it is good to have atleast one recurring donation created with UI rather than doing everything with API. That way we are covering testing of UI components in atleast one test case

Reply:
Good point I'll add back in that test now that the other ones are using the API and rename it "ui".